spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -87,7 +87,10 @@ def team_id=(team_id)
 
       response = request(:post) do |req|
         req.url "ra/v1/session/webSession"
-        req.body = { contentProviderId: team_id }.to_json
+        req.body = {
+          contentProviderId: team_id,
+          dsId: Time.now.to_i.to_s # https://github.com/fastlane/fastlane/issues/6711
```

Comment:
Of course! Glad to help. Thanks for the quick response and fast merge to fix. 

I am: https://twitter.com/summ3ra